linera-indexer/example/tests/test.rs

```diff
@@ -95,7 +95,11 @@ async fn test_end_to_end_operations_indexer(config: impl LineraNetConfig) {
     let _guard = INTEGRATION_TEST_GUARD.lock().await;
 
     let (mut net, client) = config.instantiate().await.unwrap();
-    let mut node_service = client.run_node_service(None).await.unwrap();
+    let skip_process_inbox = false;
```

Comment:
This is not really the way in Rust. If you feel bad about a boolean argument, define an enum.